### PR TITLE
Feature/lesson4

### PR DESCRIPTION
**Pull request by jsposato**
Brandon,

I'm getting an invalid DSN PDOException, that I haven't been able to track down.  Can you take a look?

### CODE REVIEW

**Comment by jsposato** [comment]
in my DiConfig, I had a key of Jsposato\Dbal\AbstractDB instead of Jsposato\Dbal\AbstractDb.  Bit by the capital 'B'.

**Comment by jsposato** [comment]
I am getting a PDOException, Invalid DSN when trying to access User functions.  I've looked up and down and can't find the problem, can you take a look?

File: config/diconfig.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    'commentModel' => $di->lazyNew( 'Jsposato\Model\Comment' )
+];
+
+$di->params['Jsposato\Controller\Index'] = [
+    'storyModel' => $di->lazyNew( 'Jsposato\Model\Story' )
+];
+
+$di->params['Jsposato\Controller\UserController'] = [
+    'config' => $config
+];
```
Comment (brandonsavage) [inline config/diconfig.php:38]: This file is probably unnecessary if you're using the configuration objects.

File: src/Controller/CommentController.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    private $commentModel;
+    protected $db;
+
+    public function __construct(CommentModel $comment) {
+        $this->commentModel = $comment;
+    }
+    
+    public function create() {
+        if(!isset($_SESSION['AUTHENTICATED'])) {
+            die('not auth');
```
Comment (brandonsavage) [inline src/Controller/CommentController.php:18]: Debugging code left in?

File: src/MasterController.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    public function __construct(Container $container, array $config = [], Router $router) {
+        $this->container = $container;
+        $this->config = $config;
+        $this->router = $router;
+    }
+
+
+    public function execute() {
+//        $call = $this->_determineControllers();
```
Comment (brandonsavage) [inline src/MasterController.php:36]: Version control mean never having to comment code out and leave it. :)

File: src/Model/Comment.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    /**
+     * getComments
+     *
+     * get all comments for a story
+     *
+     * @param $story_id
+     *
+     * @return mixed
+     *
+     * @author  jsposato
```
Comment (brandonsavage) [inline src/Model/Comment.php:55]: Standard practice doesn't include @author and @version on method calls, just at the class and file level.

**Comment by brandonsavage** [comment]
This is great! Good work! You're definitely ahead of the class. :)

**Comment by brandonsavage** [comment]
BTW, you made this PR against MY fork, not YOUR fork.I'm going to close it, but make it against your fork.

**Pull request closed without merging**